### ISSUE TITLE
jsx-element triggers rule

### ISSUE THREAD

**Issue by alfredvaa** [jsx-eslint/eslint-plugin-jsx-a11y#657]
```
<MyElement onMouseOver={() => {}} />
```
The example above is triggering the rule `jsx-a11y/mouse-events-have-key-events` because it is missing onFocus. But should this rule be applied to custom jsx-elements? I thought it was supposed to only handle native html elements like `div`.

**Comment by ljharb** [comment]
No, it definitely should not apply to custom elements.
